components/notebook-controller/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/kubeflow/kubeflow/components/notebook-controller
 
-go 1.21
+go 1.22
 
-toolchain go1.21.9
+toolchain go1.22.9
```

Comment:
shall we move it to 1.22.12 ?
https://go.dev/doc/devel/release#go1.22.0

Reply:
neglect this, seems like 1.22.9 is the latest in
https://catalog.redhat.com/software/containers/ubi8/go-toolset/5ce8713aac3db925c03774d1?gs&q=go-t